### PR TITLE
Polymer 3 Conversion

### PR DESCRIPTION
**Pull request by jeroach**

### CODE REVIEW

File: demo/d2l-navigation-button.html [hunk 1]
```diff
@@ -4,96 +4,135 @@
...
+function CycleLanguage() {
+	if (currLang >= langCycle.length - 1) {
+		currLang = 0;
+	} else {
+		currLang++;
+	}
+	document.getElementsByTagName('html')[0].setAttribute('lang', langCycle[currLang]);
+}
+function AssignClickEvent(className, handler) {
+	var elems = document.getElementsByClassName(className);
```
Comment (svanherk) [inline demo/d2l-navigation-button.html]: Only potential issue I see is around this function.  Previously we've needed to wrap calls to elements in an `window.RequestAnimationFrame`, but maybe the `window.onLoad` happens late enough?  Does this stuff still work in the demo?

**Comment by svanherk** [comment]
@dlockhart So the plan was to get Josh to try out the conversion and to leave this branch up in case you or others want to test with the Polymer 3 nav, but we were hoping to get the immersive nav components in before doing the actual merged conversion, since we are adding the immersive nav to legacy pages now.  Let us know if this is an issue!

**Comment by dlockhart** [comment]
Sounds like a plan!

**Comment by jeroach** [comment]
Closing PR for now, but leaving branch.

Will update once `d2l-immersive-nav` is ready as a hybrid component

**Pull request closed without merging**